package.json

```diff
@@ -29,7 +29,7 @@
     "dygraphs": "2.1.0",
     "emoji-dictionary": "1.0.11",
     "emojilib": "3.0.4",
-    "exports-loader": "3.1.0",
+    "exports-loader": "1.1.1",
```

Comment:
the 3.1.0 caused conflicts in the migration scripts. We recently updated it from 1.1.1 and seems to work for the other tests on the front end. So...reverting it back to 1.1.1